tests/robot-tests/tests/admin/bau/prerelease.robot

```diff
@@ -141,8 +141,8 @@ Validate prerelease has not started
     user checks nth breadcrumb contains    1    Home
     user checks nth breadcrumb contains    2    Pre-release access
 
-    ${tomorrow}=    get current datetime    %Y-%m-%dT00:00:00    1
-    ${day_after_tomorrow}=    get current datetime    %Y-%m-%dT%H:%M:%S    2
+    ${tomorrow}=    get current datetime    %Y-%m-%dT00:00:00    1    Europe/London
+    ${day_after_tomorrow}=    get current datetime    %Y-%m-%dT%H:%M:%S    2    Europe/London
 
     ${time_start}=    format uk to local datetime    ${tomorrow}    %-d %B %Y at %H:%M
     ${time_end}=    format uk to local datetime    ${day_after_tomorrow}    %-d %B %Y
```

Comment:
I am not sure we need this here. It looks like L147-8 already handle the translation?

Reply:
I wondered the same myself, but I --think-- it's safer like this. My reasoning is this: 

PRA Start comes from the backend as Utc, and the cron-job is adjusted twice a year to match BST/GMT. So on the Pre-Prod environment during summer this will be 23:00 on the day before, but during winter it will be 00:00 on the day of. 
Because we're calculating the date only in the line `${tomorrow}=    get current datetime    %Y-%m-%dT00:00:00`, this means that in summer we'll get two different dates if we use Utc or BST. 

Also, the test is currently the test is failing with the error "Page should have contained text Pre-release access will be available from 2 April 2024 at 23:00", but of course that's wrong. It's expecting 23:00, but it should be expecting 00:00 on 3 April. 

But I might have made some errors of logic somewhere along those lines!